src/libraries/Microsoft.PowerFx.Core/IR/Nodes/CallNode.cs

```diff
@@ -75,23 +75,19 @@ public bool IsLambdaArg(int i)
         }
 
         public override string ToString()
```

Comment:
I'd keep the use of parenthesis for calls

Reply:
Discussed in another comment.